Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```diff
@@ -0,0 +1,1108 @@
+category: Authentication
+commonfields:
+  id: Thycotic
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'true'
+  display: Turns on synchronization of Secrets with XSOAR credential store.
+  hidden: false
+  name: isFetchCredentials
+  required: false
+  type: 8
+- display: A list secret names for fetch credentials (separated by commas). The list
+    of names of secrets of that will be updated by the isFetchCredentials parameter
+    checked. If the list is empty , then no secrets will be updated.
```

Comment:
I think we can make this description easier to understand.
Can you please explain this parameter?

Reply:
@guykeller 

If the isFetchCredential checkbox is set, then the list of secret names for synchronization, separated by commas, is taken from this field. Each of the secrets contains the necessary information for authentication.
Synchronized data goes into Credential records.